extras/nginx-mtls-auth/README.md

```diff
@@ -23,11 +23,36 @@ WARNING: Please follow [security considerations for containers](../../docs/secur
 ## Quick Start
 
 1. Ensure you have `openvino/model_server` image available. This could be one from official releases or a local one.
+
+```bash
+docker pull openvino/model_server:latest
+```
+
 2. Run `./build.sh` to build NGINX image extra layer.
+
+```bash
+./build.sh
+```
+
 3. Run `./generate_certs.sh`  script. It will generate self-signed certificates (for testing only - follow your organization process for requesting and generating the server and client certificates).
+
+```bash
+./generate_certs.sh
+```
+
 3. In terminal 1, execute `./start_secure_model_server.sh` script. It will download sample model and start the container.
+
+```bash
```

Comment:
This will work with automated documentation tests? This md assumes splitting work into 2 terminals, which also could achieved by putting this into background with `nohup` or `&` but the way it is described here it should be easier to follow.

Reply:
I had to make semi-automated test for it. It still reads commands from documentation but I need need to make some changes in them (like port number) because everything is in bash scripts. As for two terminals, I just run docker in detached mode so I could read container id.